migrations/3_create_maintenance_activities_down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE maintenance_activities;
```

Comment:
DROP constraints as well which are created in up.sql . make changes in other files as well